src/Modules/Admin/Commands.Admin.Test/SwitchPowerBIEncryptionKeyTests.cs

```diff
@@ -82,7 +82,7 @@ public void RotatePowerBIEncryptionKey_WithAllValidParameters()
             tenantKeys.Add(tenantKey2);
```

Comment:
Another case where you can simplify by using object/collection initializer syntax.